### PR TITLE
fix(shell): position side panels above the center content

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #6658

## Summary

- position side panels above the center content
- add screenshot test

### CODE REVIEW

**Review by jcfranco — APPROVED** [review]
✨🚀👨‍🚀🪐✨

File: packages/calcite-components/src/components/shell/shell.stories.ts [hunk 1]
```diff
@@ -1643,3 +1643,19 @@ export const panelTopFloatVertical_TestOnly = (): string =>
       </calcite-panel>
     </calcite-shell-panel>
   </calcite-shell>`;
+
+export const resizeHandlePositioning = (): string =>
```
[the remark this answers is not in the document]
Comment (driskull) [reply packages/calcite-components/src/components/shell/shell.stories.ts:1647]: yeah sounds good

**Comment by driskull** [comment]
@jcfranco @macandcheese an issue I'm seeing is that if we make the panels a higher zindex than the center content, the existing tip manager would no longer be able to overflow and go outside of its container like it does in this screenshot

https://www.chromatic.com/test?appId=6266d45509d7eb004aa254fb&id=66ac0cccb9dc30b9f7b7d08e

What do you think?

**Comment by macandcheese** [comment]
> What do you think?

I think that's an expected result of the change - the Shell Panel shouldn't behave like that... I think a Dialog with `placement="bottom"` would replicate that use case if needed?

**Comment by driskull** [comment]
Ok, do you think this would be a breaking change though? Because currently we have styles specifically to tip-manager which places it absolutely above everything else. Should we maybe hold off on this one until sept main release?

**Comment by macandcheese** [comment]
I'm comfortable holding off for the breaking change release - we have Tip Manager slated for removal in v4 - but the breaking change notice should be sufficient?

**Pull request merged**